container/src/container/array_dict.py

```diff
@@ -9,196 +9,194 @@ def isiterable(obj):
         return False
 
 class array_dict:
```

Comment:
Also inherit from object:
Otherelse, in Python 2.x the class is an old-style class.
See: http://docs.python.org/release/2.2.3/whatsnew/sect-rellinks.html